pkg/clusteragent/admission/mutate/cwsinstrumentation/cws_instrumentation_test.go

```diff
@@ -11,11 +11,12 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"github.com/DataDog/datadog-agent/pkg/clusteragent/admission/mutate/common"
 	"path/filepath"
 	"strings"
 	"testing"
 
+	"github.com/DataDog/datadog-agent/pkg/clusteragent/admission/mutate/common"
```

Comment:
Normally we place `Datadog/datadog-agent` dependencies in the third block of dependencies: 1)stdlib, 2) external dependencies, 3) datadog-agent. So this one should go in the end with `github.com/DataDog/datadog-agent/pkg/config`.

It's a bit unfortunate that this rule is not enforce by the CI.